konrad/surface.py

```diff
@@ -44,12 +44,12 @@ def __init__(self, albedo=0.2, temperature=288., height=0.):
         Parameters:
             albedo (float): Surface albedo. The default value of 0.2 is a
                 decent choice for clear-sky simulation in the tropics.
-            temperature (float): Surface temperature [K].
-            height (float): Surface height [m].
+            temperature (int / float): Surface temperature [K].
+            height (int / float): Surface height [m].
         """
         self.albedo = albedo
         self.height = height
-        self['temperature'] = (('time',), np.array([temperature]))
+        self['temperature'] = (('time',), np.array([float(temperature)]))
```

Comment:
One could have used
```python
np.array([temperature], dtype=float)
```
but I can't find a single reason except for personal taste and a meaningless performance improvement 😅